pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
```

Comment:
> note: By the time I redirect buildLogs as it is right now, there is no newBuild.Name set.

Which means that this code is incorrect as is. If I am going this way, I should kick off the goroutine after I have the newBuild created so I can access its name.